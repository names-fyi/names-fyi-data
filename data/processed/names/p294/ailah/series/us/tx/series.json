[{"y":2014,"t":5,"m":0,"f":5,"mr":null,"fr":3216,"mp":0,"fp":0.00003000390050706592},{"y":2020,"t":6,"m":0,"f":6,"mr":null,"fr":2736,"mp":0,"fp":0.000039318995006487636}]