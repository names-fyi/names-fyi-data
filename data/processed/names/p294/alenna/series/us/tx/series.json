[{"y":2015,"t":9,"m":0,"f":9,"mr":null,"fr":2211,"mp":0,"fp":0.00005308419152776303},{"y":2016,"t":5,"m":0,"f":5,"mr":null,"fr":3298,"mp":0,"fp":0.00002991790526794476},{"y":2019,"t":5,"m":0,"f":5,"mr":null,"fr":3179,"mp":0,"fp":0.00003159078559965629},{"y":2021,"t":5,"m":0,"f":5,"mr":null,"fr":3228,"mp":0,"fp":0.00003225411078641973}]