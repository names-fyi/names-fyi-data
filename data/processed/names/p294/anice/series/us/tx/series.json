[{"y":1923,"t":5,"m":0,"f":5,"mr":null,"fr":955,"mp":0,"fp":0.00008587524045067326}]