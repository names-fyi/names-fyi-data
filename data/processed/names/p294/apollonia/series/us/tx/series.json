[{"y":1985,"t":5,"m":0,"f":5,"mr":null,"fr":1894,"mp":0,"fp":0.00003749559426767355},{"y":1986,"t":6,"m":0,"f":6,"mr":null,"fr":1676,"mp":0,"fp":0.00004557192769254139},{"y":1991,"t":5,"m":0,"f":5,"mr":null,"fr":2090,"mp":0,"fp":0.000036658772810921384}]