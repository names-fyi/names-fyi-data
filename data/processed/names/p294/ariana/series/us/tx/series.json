[{"y":1974,"t":6,"m":0,"f":6,"mr":null,"fr":1303,"mp":0,"fp":0.00006285355122564425},{"y":1975,"t":5,"m":0,"f":5,"mr":null,"fr":1500,"mp":0,"fp":0.00005224059930415522},{"y":1977,"t":6,"m":0,"f":6,"mr":null,"fr":1427,"mp":0,"fp":0.00005971931919976112},{"y":1978,"t":7,"m":0,"f":7,"mr":null,"fr":1304,"mp":0,"fp":0.00006770349736923553},{"y":1979,"t":6,"m":0,"f":6,"mr":null,"fr":1512,"mp":0,"fp":0.00005399713815167796},{"y":1980,"t":32,"m":0,"f":32,"mr":null,"fr":543,"mp":0,"fp":0.0002669246938707417},{"y":1981,"t":25,"m":0,"f":25,"mr":null,"fr":638,"mp":0,"fp":0.00020203979375777852},{"y":1982,"t":30,"m":0,"f":30,"mr":null,"fr":582,"mp":0,"fp":0.00023092021706500404},{"y":1983,"t":28,"m":0,"f":28,"mr":null,"fr":592,"mp":0,"fp":0.00021823680251907625},{"y":1984,"t":24,"m":0,"f":24,"mr":null,"fr":663,"mp":0,"fp":0.00018403637785735646},{"y":1985,"t":40,"m":0,"f":40,"mr":null,"fr":485,"mp":0,"fp":0.0002999647541413884},{"y":1986,"t":31,"m":0,"f":31,"mr":null,"fr":575,"mp":0,"fp":0.0002354549597447972},{"y":1987,"t":46,"m":0,"f":46,"mr":null,"fr":438,"mp":0,"fp":0.0003569904155834077},{"y":1988,"t":47,"m":0,"f":47,"mr":null,"fr":426,"mp":0,"fp":0.0003663717503995011},{"y":1989,"t":77,"m":0,"f":77,"mr":null,"fr":289,"mp":0,"fp":0.000577319587628866},{"y":1990,"t":81,"m":0,"f":81,"mr":null,"fr":285,"mp":0,"fp":0.0005887483645878761},{"y":1991,"t":129,"m":0,"f":129,"mr":null,"fr":196,"mp":0,"fp":0.0009457963385217716},{"y":1992,"t":103,"m":0,"f":103,"mr":null,"fr":243,"mp":0,"fp":0.0007487097477647743},{"y":1993,"t":131,"m":0,"f":131,"mr":null,"fr":205,"mp":0,"fp":0.0009560996971134547},{"y":1994,"t":145,"m":0,"f":145,"mr":null,"fr":188,"mp":0,"fp":0.001066419551515419},{"y":1995,"t":160,"m":0,"f":160,"mr":null,"fr":176,"mp":0,"fp":0.0011726680396654963},{"y":1996,"t":184,"m":0,"f":184,"mr":null,"fr":154,"mp":0,"fp":0.001318608867644635},{"y":1997,"t":212,"m":0,"f":212,"mr":null,"fr":147,"mp":0,"fp":0.0015155089465068233},{"y":1998,"t":219,"m":0,"f":219,"mr":null,"fr":145,"mp":0,"fp":0.0015268346417167478},{"y":1999,"t":234,"m":0,"f":234,"mr":null,"fr":140,"mp":0,"fp":0.0015895443306252208},{"y":2000,"t":339,"m":0,"f":339,"mr":null,"fr":100,"mp":0,"fp":0.0022131837856541295},{"y":2001,"t":339,"m":0,"f":339,"mr":null,"fr":101,"mp":0,"fp":0.002212678189128505},{"y":2002,"t":353,"m":0,"f":353,"mr":null,"fr":95,"mp":0,"fp":0.002263720196488348},{"y":2003,"t":374,"m":0,"f":374,"mr":null,"fr":89,"mp":0,"fp":0.002366190054409718},{"y":2004,"t":453,"m":0,"f":453,"mr":null,"fr":66,"mp":0,"fp":0.002844280359396485},{"y":2005,"t":446,"m":0,"f":446,"mr":null,"fr":61,"mp":0,"fp":0.0027773626264135905},{"y":2006,"t":508,"m":0,"f":508,"mr":null,"fr":52,"mp":0,"fp":0.003057680616832893},{"y":2007,"t":488,"m":0,"f":488,"mr":null,"fr":58,"mp":0,"fp":0.002885610383466872},{"y":2008,"t":458,"m":0,"f":458,"mr":null,"fr":64,"mp":0,"fp":0.0027317841294078353},{"y":2009,"t":442,"m":0,"f":442,"mr":null,"fr":62,"mp":0,"fp":0.00265336382900811},{"y":2010,"t":416,"m":0,"f":416,"mr":null,"fr":63,"mp":0,"fp":0.0026071860565684167},{"y":2011,"t":444,"m":0,"f":444,"mr":null,"fr":55,"mp":0,"fp":0.00284999037165415},{"y":2012,"t":449,"m":0,"f":449,"mr":null,"fr":56,"mp":0,"fp":0.0028379463128820007},{"y":2013,"t":566,"m":0,"f":566,"mr":null,"fr":37,"mp":0,"fp":0.0035182595182595183},{"y":2014,"t":712,"m":0,"f":712,"mr":null,"fr":25,"mp":0,"fp":0.004272555432206187},{"y":2015,"t":660,"m":0,"f":660,"mr":null,"fr":30,"mp":0,"fp":0.0038928407120359556},{"y":2016,"t":619,"m":0,"f":619,"mr":null,"fr":30,"mp":0,"fp":0.0037038366721715613},{"y":2017,"t":535,"m":0,"f":535,"mr":null,"fr":37,"mp":0,"fp":0.0033508706000250532},{"y":2018,"t":527,"m":0,"f":527,"mr":null,"fr":35,"mp":0,"fp":0.003321337862621399},{"y":2019,"t":555,"m":0,"f":555,"mr":null,"fr":33,"mp":0,"fp":0.0035065772015618482},{"y":2020,"t":456,"m":0,"f":456,"mr":null,"fr":48,"mp":0,"fp":0.0029882436204930604},{"y":2021,"t":448,"m":0,"f":448,"mr":null,"fr":50,"mp":0,"fp":0.0028899683264632077},{"y":2022,"t":439,"m":0,"f":439,"mr":null,"fr":50,"mp":0,"fp":0.0027098765432098763}]