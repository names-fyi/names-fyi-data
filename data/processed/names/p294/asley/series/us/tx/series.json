[{"y":1983,"t":7,"m":0,"f":7,"mr":null,"fr":1469,"mp":0,"fp":0.00005455920062976906},{"y":1984,"t":5,"m":0,"f":5,"mr":null,"fr":1837,"mp":0,"fp":0.00003834091205361593},{"y":1985,"t":7,"m":0,"f":7,"mr":null,"fr":1534,"mp":0,"fp":0.000052493831974742965},{"y":1986,"t":6,"m":0,"f":6,"mr":null,"fr":1679,"mp":0,"fp":0.00004557192769254139},{"y":1987,"t":7,"m":0,"f":7,"mr":null,"fr":1529,"mp":0,"fp":0.000054324628458344654},{"y":1989,"t":7,"m":0,"f":7,"mr":null,"fr":1626,"mp":0,"fp":0.000052483598875351456},{"y":1991,"t":6,"m":0,"f":6,"mr":null,"fr":1872,"mp":0,"fp":0.00004399052737310566}]