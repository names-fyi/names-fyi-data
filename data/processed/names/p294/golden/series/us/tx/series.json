[{"y":1910,"t":5,"m":0,"f":5,"mr":null,"fr":533,"mp":0,"fp":0.0002588661661920787},{"y":1914,"t":5,"m":0,"f":5,"mr":null,"fr":749,"mp":0,"fp":0.00014138672095916752},{"y":1916,"t":5,"m":0,"f":5,"mr":null,"fr":890,"mp":0,"fp":0.00011100503962879914},{"y":1919,"t":6,"m":0,"f":6,"mr":null,"fr":841,"mp":0,"fp":0.00011569610489780177},{"y":1922,"t":6,"m":0,"f":6,"mr":null,"fr":872,"mp":0,"fp":0.00010177426468093768},{"y":1925,"t":5,"m":0,"f":5,"mr":null,"fr":1015,"mp":0,"fp":0.00008102283223412357},{"y":1927,"t":5,"m":0,"f":5,"mr":null,"fr":1003,"mp":0,"fp":0.00008254502831294471},{"y":2019,"t":12,"m":5,"f":7,"mr":2513,"fr":2647,"mp":0.000028705936387644965,"fp":0.00004422709983951881},{"y":2021,"t":7,"m":7,"f":0,"mr":2069,"fr":null,"mp":0.000041026602821458085,"fp":0}]